src/ppx/command_line_parser.cpp

```diff
@@ -118,6 +118,12 @@ std::optional<CommandLineParser::ParsingError> CommandLineParser::Parse(int argc
             }
             mOpts.standardOptions.frame_count = opt.GetValueOrDefault<int>(-1);
         }
+        else if (opt.GetName() == "run-time-ms") {
+            if (!opt.HasValue()) {
+                return std::string("Command-line option --run-time-ms requires a parameter");
+            }
+            mOpts.standardOptions.run_time_ms = opt.GetValueOrDefault<int>(-1);
```

Comment:
default should be 0 then

Reply:
Yeah, fair. Done. Some of the other -1 values in this file probably ought to be reexamined at some point.